tests/functional/conversion/test_encode_structs.py

```diff
@@ -44,7 +44,7 @@ class SimilarStruct(Struct):
 
 EXPECTED = HexBytes(
     "0000000000000000000000000000000000000000000000000000000000000001"
-    "0200000000000000000000000000000000000000000000000000000000000000"
```

Comment:
@fubuloubu I am surprised we had so many tests expecting right padded. Surely that can't be correct. I maybe just glossed over this. But I want to make sure...

Reply:
OK so I realize dynamic sized types are right padded, but bytes32 is fixed size so it should remain left padded I think... however, I want to make sure dynamic sizes still work...